src/v3/src/components/Divider/Divider.tsx

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { Divider as MuiDivider } from '@mui/material';
+import { FunctionComponent, h } from 'preact';
+
+const Divider: FunctionComponent = () => (
+  <MuiDivider />
+);
+
+export default Divider;
```

Comment:
Do we need to consider Theming on this component? @shawyu-okta

Reply:
I don't think it will be necessary right now but I do think the odyssey team should add theme support so that the line will match the theme gray color we want to use everywhere. @leemchale-okta can you please file a ticket to the Odyssey team to eventually support theming of the divider and add the JIRA here in a comment?

Reply:
Yep will do

Reply:
https://oktainc.atlassian.net/browse/OKTA-522717

Reply:
Actually it looks like they already have the Divider on their list of components to theme: https://oktainc.atlassian.net/browse/OKTA-522665

Reply:
Oh nice, okay then we can close our ticket.